src/field_theory/abel_ruffini.lean

```diff
@@ -372,6 +372,30 @@ begin
   { exact λ α n, induction3 },
 end
 
+/-- An irreducible polynomial with an `is_solvable_by_rad` root has solvable Galois group -/
+lemma is_solvable' {α : E} {q : polynomial F} (q_irred : irreducible q)
+  (q_aeval : aeval α q = 0) (hα : is_solvable_by_rad F α) :
+  _root_.is_solvable q.gal :=
+begin
+  have q_leading_coeff_ne_zero : q.leading_coeff ≠ 0 := leading_coeff_ne_zero.mpr q_irred.ne_zero,
+  let p := q * C q.leading_coeff⁻¹,
+  have p_aeval : aeval (⟨α, hα⟩ : solvable_by_rad F E) p = 0 :=
+  subtype.ext ((aeval_alg_hom_apply (solvable_by_rad F E).val ⟨α, hα⟩ p).symm.trans
+    (show aeval α p = 0, by rw [aeval_mul, q_aeval, zero_mul])),
+  have p_irred : irreducible p,
+  { exact irreducible_of_associated ⟨⟨C q.leading_coeff⁻¹, C q.leading_coeff,
+      by rwa [←C_mul, inv_mul_cancel, C_1], by rwa [←C_mul, mul_inv_cancel, C_1]⟩, rfl⟩ q_irred },
+  have p_monic : p.monic,
+  { rw [monic, leading_coeff_mul, leading_coeff_C, mul_inv_cancel],
+    exact leading_coeff_ne_zero.mpr q_irred.ne_zero },
```

Comment:
I think it would be really nice if this could just be something like
```
obtain \<h1, h2, h3\> := aeval_eq_zero_and_irreducible_and_monic_mul_leading_coeff_inv
```